modules/tidb/src/main/java/org/testcontainers/containers/TiDBContainer.java

```diff
@@ -0,0 +1,125 @@
+package org.testcontainers.containers;
```

Comment:
One last requested change (which came up in an internal discussion):

Please change the package to `org.testcontainers.tidb` (`org.testcontainers.{module-name}`). Although not yet all modules follow this convention, this should help with mitigating the JMS Split Package issue going into the future.

Reply:
OK, I renamed this package and changed the related code.